src/libstd/os/openbsd/raw.rs

```diff
@@ -25,6 +25,10 @@ use os::unix::raw::{uid_t, gid_t};
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type off_t = i64;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type time_t = i64;
 
+#[unstable(feature = "pthread_t", issue = "0")]
+pub type pthread_t = uintptr_t;
```

Comment:
Given the [direction of liblibc](https://github.com/rust-lang/rfcs/blob/master/text/1291-promote-libc.md#changes-from-today) these can probably just be defined as `usize`.

I think that the pub-typedef-exposing-private-type pattern may not always be guaranteed to work in the future, so it'd be nice to avoid it for now.